src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -263,6 +267,31 @@ public Void scan(DocCommentTree tree, TreePath p) {
         return null;
     }
 
+    private  boolean isCanonicalRecordConstructor(ExecutableElement ee) {
```

Comment:
Remove extra whitespace between `private` and `boolean`.

I would expect javax.lang.model to provide this functionality. Unfortunately, I couldn't find it. On the other hand, a very similar method exists in jdk.javadoc.internal.doclets.toolkit.util.Utils. It would be nice if we could avoid any duplication.

Reply:
I don't see how we can avoid duplication at this time.

Reply:
It's not actually clear that `java.lang.model` has the info to provide this method, since the compact constructor is a syntactic form, that has disappeared by the time we get to the `ExecutableElement` for the constructor.  Sure, we could provide the method, but it's not clear we can detect the difference between the compact form and the use of explicit parameters.  I guess we could maybe put it on `Trees`.

Reply:
[JDK-8289249](https://bugs.openjdk.org/browse/JDK-8289249)